tools/azsdk-cli/Azure.Sdk.Tools.Cli/README.md

```diff
@@ -122,3 +122,13 @@ dotnet run --project Azure.Sdk.Tools.Cli -- release-plan get --work-item-id YOUR
 In either case, the _same_ code will be invoked to get both results.
 
 This server is intended to run in **local mcp mode only** and will utilize your environment cached settings to communicate where authentication is necessary.
+
+## Telemetry Configuration
+Telemetry collection is on by default.
+
+To opt out, set the environment variable AZSDKTOOLS_MCP_COLLECT_TELEMETRY to false in your environment.
+
+If you need to direct telemetry to an alternate Application Insights instance (for local testing or private collection), set one of the following environment variables in your environment or in your hosting configuration:
```

Comment:
so for testing we want it off?
we already have an env variable: https://github.com/Azure/azure-sdk-tools/tree/main/tools/azsdk-cli#test-mode

Reply:
It is per privacy team, we need to provide a way to turn it off.

Reply:
We can probably just disable the CLI exporter when in test (https://github.com/Azure/azure-sdk-tools/blob/main/tools/azsdk-cli/Azure.Sdk.Tools.Cli/Telemetry/TelemetryService.cs#L48-L50). Or we could consider injecting the exporter.